### ISSUE TITLE
YSFGateway Blocking option

### ISSUE THREAD

**Issue by w4edp** [g4klx/YSFClients#301]
Is there a way in YSFGateway to block by Call Sign, CCS7 ID, IP Address, etc?  I have an issue with a user and if I can't block them then I'll have no other choice but to shut down the Gateway since so far they have refused to cooperate.

**Comment by ChrisK9EQ** [comment]
You can try blocking their IP address. They can, of course, change CCS7 ID
and call sign. Blocking the IP address can be done in the OS.

**Comment by w4edp** [comment]
True, although IP addresses change on a regular basis since the vast majority of residential connections are DHCP based, much more so than call signs and/or CCS7 ID’s.
